### PR TITLE
allow separator colors to be updated

### PR DESCRIPTION
**Pull request by gregflynn**
I wanted to be able to cache separators in my forked list update functions for taglist and tasklist, this seemed like the lowest touch way to do it

### CODE REVIEW

File: util/separators.lua [hunk 1]
```diff
@@ -18,14 +18,24 @@ local separators = { height = 0, width = 9 }
...
     widget.fit = function(m, w, h)
         return separators.width, separators.height
     end
 
+    widget.update = function(col1, col2)
+        widget.col1 = col1
+        widget.col2 = col2
+        local vis = widget:get_visible()
+        widget:set_visible(not vis)
+        widget:set_visible(vis)
```
Comment (gregflynn) [inline util/separators.lua]: this goofiness is purely just to get the widget to redraw, if there's a better way I'd love to know

Comment (lcpz) [reply util/separators.lua]: It should be `widget:emit_signal("widget::redraw_needed")`.

**Comment by lcpz** [comment]
I need you to squash.

```shell
git rebase -i HEAD~2
# change "pick 7184149" to "f 7184149" and save
git push -u force
```

**Comment by lcpz** [comment]
I did it for you. Next time please learn how to do it.

**Comment by gregflynn** [comment]
I'm at work, was going to do it tonight... there's also a squash & merge button...

**Comment by lcpz** [comment]
> squash & merge button...

I can't find it, even though it's just what I did.

**Comment by gregflynn** [comment]
image
This is a screenshot from a PR on another repo I'm on, it makes this process much less painful

**Pull request merged**